metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/0.3.0/configuration/metron-env.xml

```diff
@@ -246,7 +246,7 @@ bolt.hdfs.file.system.url={{ default_fs }}
 bolt.hdfs.wip.file.path=/paloalto/wip
 bolt.hdfs.finished.file.path=/paloalto/rotated
 bolt.hdfs.compression.codec.class=org.apache.hadoop.io.compress.SnappyCodec
-index.hdfs.output={{ metron_apps_enrichment_dir }}
```

Comment:
I agree that this works, but could we do
`index.hdfs.output={{ metron_apps_enrichment_dir }}/indexing`
or even defined in the config, e.g.
`index.hdfs.output={{ metron_indexing_hdfs_dir }}/indexing`

I don't have a particularly strong opinion on constructing it in the params vs metron-env, but we should be using metron_apps_enrichment_dir since we already calculate it in the params anyway.